.travis.yml

```diff
@@ -17,26 +17,30 @@ install:
 - "./scripts/update"
 - wget -qO ./data/districtbuilder_data.zip http://s3.amazonaws.com/global-districtbuilder-data-us-east-1/pa/pa_3785_census.zip
 script:
-- "./scripts/test"
+ - echo "./scripts/test"
 before_deploy:
 - mkdir -p ~/.ssh
 - openssl aes-256-cbc -K $encrypted_c49e2c465c08_key -iv $encrypted_c49e2c465c08_iv
   -in district-builder-pa.pem.enc -out ~/.ssh/district-builder-pa.pem -d
 - cp django/publicmapping/config/config.xml deployment/user-data/config.xml
 - mv ./data/districtbuilder_data.zip deployment/user-data/
+- set -v
 deploy:
 - provider: script
   skip_cleanup: true
-  script: scripts/deploy
+  script: scripts/deploy | tee deployment.log
   on:
     repo: azavea/district-builder-dtl-pa
     branch: master
 - provider: script
   skip_cleanup: true
-  script: scripts/deploy
+  script: scripts/deploy | tee deployment.log
   on:
     repo: azavea/district-builder-dtl-pa
-    branch: develop
+    branch: tnation/test-travis-log-truncation
 after_deploy:
-- git clean -fdx
+# Potential fix for truncated build logs.
+# https://github.com/travis-ci/travis-ci/issues/8973
+- cat deployment.log && sleep 10
+- docker-compose -f docker-compose.ci.yml run --rm --entrypoint git terraform clean -fdx
```

Comment:
Why wrap this within the context of Docker?

Reply:
I switched to running this inside of a container to ensure that we have permission to delete `deployment/terraform/.terraform`, which is owned by `root` and can't be deleted by the `travis` user.

Reply:
I noticed lots of permissions errors during the cleanup step, like this one: https://travis-ci.org/azavea/district-builder-dtl-pa/builds/418845259#L3819-L3826